--- ddd-3.3.10/configure.ac~	2004-11-11 00:03:30.000000000 +0100
+++ ddd-3.3.10/configure.ac	2004-11-11 00:03:30.000000000 +0100
@@ -57,7 +57,7 @@
 dnl  1.x:  minimum required version
 dnl  foreign:  we don't follow the normal rules for GNU packages (no COPYING
 dnl            file in the top srcdir, etc, etc), so stop complaining.
-AM_INIT_AUTOMAKE([1.9 foreign])
+AM_INIT_AUTOMAKE([1.8.5 foreign])
 AC_PROG_MAKE_SET
 dnl
 dnl
